.gitignore

```diff
@@ -1,3 +1,3 @@
-!.gitignore
+#.gitignore
```

Comment:
- [ ] [Optional] This is optional but you can remove unused headers from your README file so it looks more professional,

![Screenshot_20220526_175356](https://user-images.githubusercontent.com/25227022/170536747-fe31dbd4-d3af-42c1-bf57-cbf44a963584.png)